### PR TITLE
Replace glob-promise with fast-glob

### PR DESCRIPTION
**Pull request by krynble**
Updating PR https://github.com/n8n-io/n8n/pull/1686 to latest.

Tested with a few workflows and credentials, all working fine.

Consideration: total package size goes from ~66kb to ~86kb.

### CODE REVIEW

**Review by BHesseldieck — APPROVED** [review]
amazing! 😊

File: packages/cli/commands/import/credentials.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ import {
 } from 'n8n-workflow';
 
 import * as fs from 'fs';
-import * as glob from 'glob-promise';
+import * as fg from 'fast-glob';
```
Comment (BHesseldieck) [inline packages/cli/commands/import/credentials.ts:24]: maybe let's keep it as `glob`, so we don't need to touch the code

```suggestion
import * as glob from 'fast-glob';
```

Comment (krynble) [reply packages/cli/commands/import/credentials.ts]: Thanks for the suggestion, @BHesseldieck

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with 0.135.0

**Pull request merged**